### ISSUE TITLE
I can't upload a profile picture from the "edit profile" view

### ISSUE THREAD

**Issue by uri-rodberg** [speedy-net/speedy-net#22]
I can't upload a profile picture from the "edit profile" view in both websites (Speedy Net / Speedy Match).

I receive an error message: "No file was submitted. Check the encoding type on the form."

**Comment by acjh** [comment]
Fixed by ec8845f:

```diff
  class CustomPhotoWidget(forms.widgets.Widget):
+     needs_multipart_form = True
```

`CustomPhotoWidget` is used by (1) `ProfileForm` and `SpeedyMatchProfileBaseForm`.
`SpeedyMatchProfileBaseForm` is subclassed by (2) `SpeedyMatchProfileActivationForm`.

(1) `ProfileForm` is rendered by:
```html
<div data-block="ProfileForm">
    {% crispy form %}
</div>
```

(2) `SpeedyMatchProfileActivationForm` is rendered by:

```html
<form action="..." method="..." class="..." enctype="multipart/form-data">
    ...
        {% for field in form %}
            ...
                {{ field }}
            ...
        {% endfor %}
    ...
</form>
```

**Comment by uri-rodberg** [comment]
I submitted a question and answer on [Stack Overflow](https://stackoverflow.com/questions/61535301/files-not-uploaded-in-django-form-encoding-type-incorrect).
